test/draw/RasterCanvasTest.ooc

```diff
@@ -105,6 +106,20 @@ RasterCanvasTest: class extends Fixture {
 			input free()
 			output free()
 		})
+		this add("resize with box", func {
+			inputFlower := "test/draw/input/Flower.png"
+			inputSpace := "test/draw/input/Space.png"
+			output := "test/draw/output/RasterCanvas_resizeWithBox.png"
+			imageFlower := RasterBgr open(inputFlower)
+			outpuImage := RasterBgr open(inputSpace)
+			RasterCanvas resizeNearestNeighbour(imageFlower buffer pointer as ColorBgr*, outpuImage buffer pointer as ColorBgr*, IntBox2D new(imageFlower size), IntBox2D new(20, 30, 100, 250), imageFlower stride, outpuImage stride, imageFlower bytesPerPixel)
```

Comment:
This will be removed later, replaced with `outpuImage canvas draw(imageFlower, IntBox2D new(imageFlower size), IntBox2D new(20, 30, 100, 250))`. I just don't want to include everything in one PR.